app/controllers/home_controller.rb

```diff
@@ -0,0 +1,4 @@
+class HomeController < ApplicationController
+  def index
+  end
+end
```

Comment:
What do you think of adding a new line at the end of the file to avoid this warning on Github. It's kind of a best practice: https://thoughtbot.com/blog/no-newline-at-end-of-file